lib/rest2.js

```diff
@@ -1303,6 +1340,55 @@ class RESTv2 {
       .then(_takeResNotify)
   }
 
+  /**
+   * Send Multiple order-related operations.
+   *
+   * @param {object} params - Multiple order-related operations parameters
+   * @param {Array.<Array.<string,object>>} params.ops - List of operations to perform
+   * @param {Array.<string,Order>} [params.ops[0]] - Order Submit ['on', new Order({...})]
+   * @param {Array.<string,{id:number}>} [params.ops[1]] - Order cancellation ['oc', { id: 1 }]
+   * @param {Array.<string,{id:Array.<number>}>} [params.ops[2]] - Multi Order Cancellation ["oc_multi", {id: [1, 2, ...]}]
+   * @param {Array.<string,Order>} [params.ops[3]] - Order Update ["ou", {id: 1, price: '15', amount: '0.001'}]
+   * @param {Function} [cb] - callback
+   * @returns {Promise} - p
+   * @see https://docs.bitfinex.com/reference#rest-auth-order-multi
+   */
+  orderMultiOp (params = { ops: [] }, cb) {
```

Comment:
Maybe this can be done with a more friendly user experience? 
Like https://github.com/bitfinexcom/bfx-api-node-rest/blob/7d05074da61487d09ed64b98a9a660dc3f6d0a63/lib/rest2.js#L1286

Reply:
Hi @ezewer. Thank you for reviewing this for me. I'm open for suggestions if you have any.  `OrderMultiOp` is a completely different beast compared to `submitOrder` which does only one thing and operates only on Order class instance.

Reply:
In order to improve user experience here, we could create a set of sub functions which call into `orderMultiOp`. For example we could also add a `submitOrderMulti(Array<Order>)` and a `updateOrderMulti(Array<map>)`

Reply:
Hey @JacobPlaster, thanks for the tip. I've broken it down a bit, kept original implementation too for flexibility. Let me know if you'd like to change something else